admin/data/sql/migrations/046.do.update-reasons.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE [mtc_admin].[attendanceCode] SET reason = 'Just arrived with EAL' where reason = 'Just arrived';
```

Comment:
this needs to be renamed as number 50

Reply:
@GeorgeChatzigiannis 46 was available and I added this entry myself =) Somebody overwrote my entry. I'll update the number